IntelliTect.TestTools.Console/ConsoleAssert.cs

```diff
@@ -137,12 +143,14 @@ private static string Expect(string expected, Action action, Func<string, string
         /// </summary>
         /// <param name="expected">Expected "view" to be seen on the console,
         /// including both input and output</param>
+        /// <param name="escapeCharacter"></param>
         /// <param name="action">Method to be run</param>
-        public static string ExpectLike(string expected, Action action)
+		[Obsolete]
+        public static string ExpectLike(string expected, char escapeCharacter, Action action)
```

Comment:
Why did you change the method signature (arguably making it a new API), then mark it as obsolete?